packages/babel-cli/src/babel/dir.js

```diff
@@ -43,18 +43,24 @@ module.exports = function (commander, filenames) {
     }
   }
 
+  let filterDotFiles = function (x) {
+    return x[0] === ".";
+  };
+  let copyFilesFilter = () => false;
+
   function handle(filename) {
     if (!fs.existsSync(filename)) return;
 
     let stat = fs.statSync(filename);
-
     if (stat.isDirectory(filename)) {
       let dirname = filename;
-
-      _.each(util.readdir(dirname), function (filename) {
+      if (commander.copyDotFiles) {
+        copyFilesFilter = filterDotFiles;
```

Comment:
Okay, what I understood was as follows:
When the `--copy-dot-files` option is passed we copy the dot files. Hence here, I'm checking if the option is true and using the filter to get *only* the dot files and copy them. In case this option isn't passed, we use the default filter which excludes the dot files. Did I misinterpret something here?